Tests/demistomock/demistomock.py

```diff
@@ -388,18 +388,19 @@ def params():
     """(Integration only)
     Retrieves the integration parameters object
 
-    :return: Integrations parameters object
-    :rtype: ``dict``
+    Returns:
+      dict: Integrations parameters object
+
     """
     return {}
 
 
 def args():
-    """
-    Retrieves a command / script arguments object
+    """Retrieves a command / script arguments object
+
+    Returns:
+      dict: Arguments object
```

Comment:
```suggestion
      dict. Arguments object
```
here and everywhere else

Reply:
both are ok actually